lib/timetable.dart

```diff
@@ -1,5 +1,11 @@
 library timetable;
 
 import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
 
 part 'src/timetable.dart';
```

Comment:
https://dart.dev/guides/libraries/create-library-packages

Please don't use part and part of.

import & export are better suited